nekoyume/Assets/_Scripts/Game/Character/ArenaCharacter.cs

```diff
@@ -492,6 +492,39 @@ public IEnumerator CoBlowAttack(IReadOnlyList<ArenaSkill.ArenaSkillInfo> skillIn
             }
         }
 
+        public IEnumerator CoDoubleAttackWithCombo(IReadOnlyList<ArenaSkill.ArenaSkillInfo> skillInfos)
+        {
+            if (skillInfos is null || skillInfos.Count == 0)
+            {
+                yield break;
+            }
+
+            var skillInfosCount = skillInfos.Count;
+            for (var i = 0; i < skillInfosCount; i++)
+            {
+                var info = skillInfos[i];
+                ActionPoint = () =>
+                {
+                    var target = info.Target.Id == Id ? this : _target;
+
+                    Vector3 effectPos = target.transform.position;
+                    effectPos.x += 0.3f;
+                    effectPos.y = Stage.StageStartPosition + 0.32f;
+
+                    var effectObj = Game.instance.Stage.objectPool.Get($"TwinAttack_0{i + 1}", false, effectPos) ??
```

Comment:
i가 10보다 커지면 어떻게 되는것인가요

Reply:
아 해당부분 리소스도 2개밖에 없어서 2가 최대임을 가정하고 작성이된부분이긴합니다.
생각해보니 혹시 모를 확장성을위해서 이렇게 바꿔도 좋을것같긴하네요~!
```suggestion
                    var effectObj = Game.instance.Stage.objectPool.Get($"TwinAttack_{i + 1:D2}", false, effectPos) ??
```